core/src/main/java/jenkins/slaves/systemInfo/SlaveSystemInfo.java

```diff
@@ -24,4 +27,14 @@
     public static ExtensionList<SlaveSystemInfo> all() {
         return ExtensionList.lookup(SlaveSystemInfo.class);
     }
+
+    /**
+     * Returns the permission required for user to see this system info extension on the "System Information" page for the Agent
```

Comment:
I would like to see more than the bare minimum of Javadoc here, see e.g. https://github.com/jenkinsci/jenkins/blob/7b654b25df94a10cfcc019ba6dda5c34219c3a61/core/src/main/java/hudson/model/AdministrativeMonitor.java#L170-L185 for comparison. Of course, this is a far simpler extension point, but at least supported (useful) values would make sense, and what the default is.